packagedb/api.py

```diff
@@ -517,11 +517,24 @@ def filter_by_checksums(self, request, *args, **kwargs):
         for field, value in data.items():
```

Comment:
I think it would be better to pop `enhance_package_data` from `data` before iterating through it, then use `enhance_package_data` afterwards.
```suggestion
        enhance_package_data = data.pop('enhance_package_data', False)
        for field, value in data.items():
```